### PR TITLE
Feature/adding database ALL TESTS GREEN

### PR DESCRIPTION
**Pull request by KasparsBelovs**

### CODE REVIEW

File: Controllers/TestingApiController.cs [hunk 1]
```diff
@@ -8,10 +9,19 @@ public class TestingApiController : ApiController
...
+                { 
+                    ctx.Flights.RemoveRange(ctx.Flights); 
+                    ctx.Airports.RemoveRange(ctx.Airports);
+                    ctx.SaveChanges();
+                }
 
-            return Ok();
+                return Ok();
+            }
+            //FlightStorage.AllFlights.Clear();
```
Comment (ausmoons) [inline Controllers/TestingApiController.cs]: ?

File: Models/FlightStorage.cs [hunk 2]
```diff
@@ -15,8 +19,13 @@ public static Flight AddFlight(Flight flight)
             {
                 flight.Id = _id;
                 _id++;
-               AllFlights.Add(flight);
-    
+               //AllFlights.Add(flight);
```
Comment (ausmoons) [inline Models/FlightStorage.cs]: ?

File: Models/FlightStorage.cs [hunk 3]
```diff
@@ -25,132 +34,171 @@ public static Flight FindFlight(int id)
         {
             lock (Locker.Lock)
             {
-                return AllFlights.FirstOrDefault(x => x.Id == id);
+                using (var ctx = new FlightPlannerDbContext())
+                {
+                    return ctx.Flights.Include(x => x.To).Include(x => x.From).SingleOrDefault(x => x.Id == id);
+                }
+                   // return AllFlights.FirstOrDefault(x => x.Id == id);
```
Comment (ausmoons) [inline Models/FlightStorage.cs]: ?

**Review by ausmoons — COMMENTED** [review]
Parskatiju, bet jasagaida Edgara approve

File: Controllers/TestingApiController.cs (hunk 1 above)
Comment (KasparsBelovs) [reply Controllers/TestingApiController.cs]: deleted useless comment

File: Models/FlightStorage.cs (hunk 2 above)
Comment (KasparsBelovs) [reply Models/FlightStorage.cs]: deleted useless comment

File: Models/FlightStorage.cs (hunk 3 above)
Comment (KasparsBelovs) [reply Models/FlightStorage.cs]: deleted useless comment

**Review by ausmoons — APPROVED** [review]

**Comment by KasparsBelovs** [comment]
done

**Pull request closed without merging**